terragrunt.hcl

```diff
@@ -3,7 +3,6 @@ generate "provider" {
   if_exists = "overwrite_terragrunt"
   contents = <<EOF
 provider "azurerm" {
-  version = "=2.46.1"
```

Comment:
removing `version` it will update the provider with latest version?

Reply:
It's not recommended anymore. A big warning is shown. I had to place the version inside the modules themselves:
https://www.terraform.io/docs/language/providers/requirements.html

https://github.com/pagopa/io-infrastructure-modules-new/pull/212/files#diff-af4cba7dbb32a265b895b22a4c6eb360fedfb73d6a33e7df36ecb4f8c2bb8254R7

Reply:
> > Also terraform modules needs to be updated to v3.0.0?
> > here an example
> > https://github.com/pagopa/io-infrastructure-modules-new/blob/master/azurerm_api_management/main.tf#L12
> 
> It's already done....there is PR that needs to be merged in modules repository.
> 
> [pagopa/io-infrastructure-modules-new#212](https://github.com/pagopa/io-infrastructure-modules-new/pull/212)

yes I miss that PR :)